project/PythonClientPostBuild.scala

```diff
@@ -164,4 +165,37 @@ object PythonClientPostBuild {
       }
     }
   }
+
+  /**
+   * Deletes the 'streams' directory from the target folder.
+   *
+   * @param log            The logger to output informational messages.
+   * @param targetDir      The target directory where 'streams' is located.
+   */
+  def deleteIrrelevantFiles(
+      log: Logger,
+      targetDir: String
+  ): Unit = {
+    // Define a sequence of paths to delete
+    val pathsToDelete: Seq[Path] = Seq(
+      Paths.get(targetDir, "streams"),
+      Paths.get(targetDir, ".github"),
+      Paths.get(targetDir, ".openapi-generator"),
+      Paths.get(targetDir, ".openapi-generator-ignore")
+    )
+
+    pathsToDelete.foreach { path =>
+      if (Files.exists(path)) {
+        log.info(s"Deleting '${path.getFileName}' at $path")
+        deleteRecursively(path) match {
+          case Success(_) =>
```

Comment:
This is code bloat. This error message could have been done within deleteRecursively function. rather all callers of this function has to write the same code to print the error/ack message.